johill

```diff
@@ -189,7 +191,10 @@ class Converter:
             kind = item['field'].lower()
             if kind == 'assignee':
                 kind = 'owner'
-                value = self.user_email(item['newvalue'])
+                try:
+                    value = self.user_email(item['newvalue'])
+                except:
+                    value = None
```

Comment:
Mh, I'd rather have `user_email()` return None if `key` is None and use `self.user_email(item.get('newvalue')` here.

This way we'd avoid repeating the `try`/`except` block everywhere. :)